test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -128,4 +128,25 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
       AccountRootFlag.checkFlag(AccountRootFlag.LSF_REQUIRE_AUTH, flags!),
     )
   })
+
+  it.only('createTrustline - valid request', async function (): Promise<void> {
+    this.timeout(timeoutMs)
+    const issuer = await XRPTestUtils.randomWalletFromFaucet()
+    await issuedCurrencyClient.createTrustline(
```

Comment:
@amiecorso this test fails, and i'm getting  

`Error: version_invalid: version bytes do not match any of the provided version(s)`

 which comes from https://github.com/ripple/ripple-address-codec/blob/0103a9c850e83d30717d883debdecf1547d4d0b9/src/xrp-codec.ts#L94

am i composing the transaction incorrectly?

given the example TrustSet at https://xrpl.org/trustset.html, i'm pretty sure that 'USD' and '0' are valid, which makes me wonder if perhaps something else is going on...is the address in the wrong format?